test/Hydration-test.js

```diff
@@ -16,4 +19,34 @@ describe('Hydration', function() {
   it('should be an instance of Hydration', function() {
     expect(hydration).to.be.an.instanceof(Hydration);
   });
+
+  describe('Hydration Records', function(){
+    beforeEach(function() {
+      user.logHydration(1, "2019/06/15", 37);
+      user.logHydration(1, "2019/06/16", 75)
+      user.logHydration(1, "2019/06/17", 42)
+      user.logHydration(1, "2018/06/18", 49)
+      user.logHydration(1, "2019/06/19", 56)
+      user.logHydration(1, "2019/06/20", 34)
+      user.logHydration(1, "2019/06/21", 65)
+      user.logHydration(1, "2019/06/22", 77)
+      user.logHydration(1, "2019/06/23", 100)
+      user.logHydration(1, "2019/06/24", 67)
+      user.logHydration(1, "2019/06/25", 84)
+      user.logHydration(1, "2019/06/26", 49)
+    })
```

Comment:
Just a thought - This looks very redundant, do we need this much data for the tests here?

Reply:
It is redundant, but I wanted to include more than 7 examples so I could test multiple dates to make sure it worked. We could reduce it to just a single test and maybe only 8 examples just to prove that it doesn't grab the whole array. We can refactor that easily down the line.